scripts-dev/complement.sh

```diff
@@ -32,6 +35,24 @@ echo_if_github() {
   fi
 }
 
+# parse our arguments
+skip_docker_build=""
+while [ $# -ge 1 ]; do
+    arg=$1
+    case "$arg" in
+        "-h")
+            echo >&2 "Usage: $0 [-f] <go test arguments>"
```

Comment:
Might be worth adding a line which tells people to consult `go help testflag` here? (I usually do `go help test` which is less helpful.)

Reply:
have added more words.

Reply:
many thanks!